{{cookiecutter.project_name}}/server/settings/components/common.py

```diff
@@ -170,11 +170,10 @@
 )
 
 PASSWORD_HASHERS = [
-    'django.contrib.auth.hashers.BCryptSHA256PasswordHasher',
-    'django.contrib.auth.hashers.BCryptPasswordHasher',
+    'django.contrib.auth.hashers.Argon2PasswordHasher',
     'django.contrib.auth.hashers.PBKDF2PasswordHasher',
     'django.contrib.auth.hashers.PBKDF2SHA1PasswordHasher',
-    'django.contrib.auth.hashers.Argon2PasswordHasher',
+    'django.contrib.auth.hashers.BCryptSHA256PasswordHasher',
```

Comment:
I think that this hasher won't work without `bcrypt` package 🤔

Reply:
It's enabled by default though. See [docs](https://docs.djangoproject.com/en/3.2/ref/settings/#std:setting-PASSWORD_HASHERS).